mevshare/builders.go

```diff
@@ -70,9 +73,15 @@ func LoadBuilderConfig(file string) (BuildersBackend, error) {
 			return BuildersBackend{}, err
 		}
 
+		cl := jsonrpc.NewClientWithOpts(builder.URL, &jsonrpc.RPCClientOpts{
+			HTTPClient:         nil,
+			CustomHeaders:      map[string]string{orderflowHeader: flashbotsSource},
```

Comment:
suggestion:

make the inclusion of the `orderflow` header a flag (eg. `orderflowHeader`), with a default of `false`.

Reply:
Thanks for suggestions! Builder-related info is configured via file though, so I made it configurable there